README.md

```diff
@@ -72,6 +72,27 @@ populated if you simply run `go build`. You have to pass in command
 line flags as defined in `Makefile.INCLUDE` (see `${BUILDFLAGS}`) to
 do that.
 
+## Release checklist
+
+To cut a new release of Prometheus, follow this checklist:
+
+1. Consider updating all or some of the vendored dependencies before the
+   release. Make sure that https://github.com/prometheus/client_golang is
+   vendored at a release version.
+1. Create a PR that updates the `CHANGELOG.md` along with the `version/VERSION` file and get it merged.
+1. Run `make tag` to tag the new release and push it to GitHub.
+1. Head to https://github.com/prometheus/prometheus/releases and create a new release for the pushed tag.
+1. Attach the relevant changelog excerpt to the release on GitHub.
+1. Build `linux-amd64`, `linux-386`, and `darwin-amd64` binaries via `make tarball` and attach them to the release.
+1. Update the `stable` branch to point to the new release (to tag the latest release image on https://registry.hub.docker.com/u/prom/prometheus/):
```

Comment:
Which reminds me: We have the stable branch so that we could backport bugfixes from master if needed, right? But if that happens, we would cut a new release from the stable branch and not from master. That procedure should be described, too.